### PR TITLE
feat(ui): enable wrap long lines in code block

### PR DESCRIPTION
**Pull request by liangfung**
### Change
Add a button to enable/disable line wrapping after stream completion.

### Screen record
https://jam.dev/c/4860fbc8-1c85-48e6-8fe3-b3fa1919201a

### CODE REVIEW

**Comment by wsxiaoys** [comment]
Use https://lucide.dev/icons/wrap-text  / https://lucide.dev/icons/align-justify for toggling line wrapping

File: ee/tabby-ui/components/ui/codeblock.tsx [hunk 1]
```diff
@@ -83,6 +91,27 @@ const CodeBlock: FC<Props> = memo(
...
+                    onClick={() => setWrapLongLines(!wrapLongLines)}
+                  >
+                    {wrapLongLines ? <IconAlignJustify /> : <IconWrapText />}
+                  </Button>
+                </TooltipTrigger>
+                <TooltipContent>
+                  <p className="m-0">
+                    {wrapLongLines
+                      ? 'Disable line wrapping'
+                      : 'Enable line wrapping'}
```
Comment (wsxiaoys) [inline ee/tabby-ui/components/ui/codeblock.tsx:110]:
```suggestion
Toggle word wrap
```

File: ee/tabby-ui/app/search/components/search.tsx [hunk 2]
```diff
@@ -1135,6 +1136,7 @@ function AnswerBlock({
           onCodeCitationMouseLeave={onCodeCitationMouseLeave}
           contextInfo={contextInfo}
           fetchingContextInfo={fetchingContextInfo}
+          canWrapLongLines={isLoading}
```
Comment (wsxiaoys) [inline ee/tabby-ui/app/search/components/search.tsx:1139]: should be `!isLoading`?

Comment (liangfung) [reply ee/tabby-ui/app/search/components/search.tsx]: done

**Pull request merged**